scalastyle-config.xml

```diff
@@ -48,15 +48,9 @@
             <parameter name="maxParameters"><![CDATA[8]]></parameter>
         </parameters>
     </check>
-    <check level="error" class="org.scalastyle.scalariform.MagicNumberChecker" enabled="false">
-        <parameters>
-            <parameter name="ignore"><![CDATA[-1,0,1,2,3]]></parameter>
-        </parameters>
-    </check>
     <check level="error" class="org.scalastyle.scalariform.NoWhitespaceBeforeLeftBracketChecker" enabled="true"></check>
     <check level="error" class="org.scalastyle.scalariform.NoWhitespaceAfterLeftBracketChecker" enabled="true"></check>
-    <check level="error" class="org.scalastyle.scalariform.ReturnChecker" enabled="true"></check>
-    <check level="error" class="org.scalastyle.scalariform.NullChecker" enabled="false"></check>
+    <check customId="return" level="error" class="org.scalastyle.scalariform.ReturnChecker" enabled="true"></check>
```

Comment:
why do we need a `customId` property here?

Reply:
For disabling this check, which would be helpful for t-b-v.
We have some complex functions there that would require refactoring in order to pass this - I'd opt for disabling this check in some places at first.